src/python/pants_release/generate_github_workflows.py

```diff
@@ -1259,6 +1259,19 @@ def release_jobs_and_inputs() -> tuple[Jobs, dict[str, Any]]:
                         """
                     ),
                 },
+                {
+                    "name": "Trigger docs sync",
+                    "env": {
+                        "GH_TOKEN": "${{ secrets.WORKER_PANTS_PANTSBUILD_ORG_TRIGGER_PAT }}",
+                    },
+                    "run": dedent(
+                        """\
+                        RELEASE_TAG=${{ steps.get_info.outputs.build-ref }}
+                        RELEASE_VERSION="${RELEASE_TAG#release_}"
+                        gh workflow run sync_docs.yml -F "version=$RELEASE_VERSION" -R pantsbuild/pantsbuild.org
```

Comment:
No `reviewer` as nothing in this workflow had the releaser AFAICT. `git show <tag>` would give the email of the tagger, but that isn't the "handle" as required by `gh pr create --reviewer`.

Maybe we can just always use @benjyw ? :joy:

Reply:
Let's not...

Reply:
Ah hm, the GitHub UI does show the person who pushed the tag connected to the action run.

![image](https://github.com/pantsbuild/pants/assets/1203825/78596638-5da9-43fb-a227-71f6fa3ba6e3)

I wonder if `${{ github.actor }}` is the right thing?

```suggestion
                        gh workflow run sync_docs.yml -F "version=$RELEASE_VERSION" -F "reviewer=${{ github.actor }}" -R pantsbuild/pantsbuild.org 
```

Reply:
I'm feeling confident. Let's give it a whirl